### PR TITLE
Changes for a reindex on 2021-01-08

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/platform/issues/4948

### CODE REVIEW

File: pipeline/terraform/stack/service_id_minter.tf [hunk 1]
```diff
@@ -77,7 +77,7 @@ module "id_minter" {
 module "id_minter_topic" {
   source = "../modules/topic"
 
-  name       = "${local.namespace_hyphen}_work_id_minter"
+  name       = "${local.namespace_hyphen}_id_minter_output"
```
Comment (jamesgorrie) [inline pipeline/terraform/stack/service_id_minter.tf:80]: Why this change?

File: pipeline/terraform/stack/service_merger.tf [hunk 2]
```diff
@@ -7,7 +7,7 @@ module "merger_queue" {
   alarm_topic_arn = var.dlq_alarm_arn
 
   # This has to be longer than the flush interval in the merger
```
Comment (jamesgorrie) [inline pipeline/terraform/stack/service_merger.tf:9]:
```suggestion
  # This has to be longer than the `flush_interval_seconds` in the merger
```
❓

**Review by jamesgorrie — APPROVED** [review]
Really nice clear comments on why the changes are being made!

File: pipeline/terraform/stack/service_id_minter.tf (hunk 1 above)
Comment (alexwlchan) [reply pipeline/terraform/stack/service_id_minter.tf:80]: The old name comes from a time when we had separate ID minters for images and works; that’s no longer the case.

And although “output” is implicit from this being SNS, we do have topics that aren’t the output of any app and being explicit about that is useful.

**Review by jamesgorrie — APPROVED** [review]
